src/app/WriteClient.h

```diff
@@ -47,7 +47,7 @@ class InteractionModelEngine;
  *  @brief The read client represents the initiator side of a Write Interaction, and is responsible
  *  for generating one Write Request for a particular set of attributes, and handling the Write response.
  *  Consumer can allocate one write client, then call PrepareAttribute, insert attribute value, followed by FinishAttribute for
- * every attribute it wants to insert in write request, then call SendWriteRequest
+ * every attribute it wants to insert in write request, then call SendWriteRequestMessage
```

Comment:
But with chunking it would not be a single write request message anyway....

Reply:
Rename it back, thanks